server/utils.js

```diff
@@ -1,7 +1,10 @@
 import { join } from 'path'
 import { readdirSync, existsSync } from 'fs'
 
-export function getChunkNameFromFilename (filename) {
+export function getChunkNameFromFilename (filename, dev) {
+  if (dev) {
+    return filename.replace(/.[^.]*$/, '')
```

Comment:
huh, is this because webpack handles naming different based on `NODE_ENV`?

Reply:
Yep, with chunk hash or without, but I didn't change this. It's from another PR.